examples/ota_https/main/app_main.c

```diff
@@ -46,9 +46,12 @@ esp_err_t wifi_init_connect(void)
 
     esp_err_t err;
     err = esp_event_loop_create_default();
-    if (err == ESP_ERR_INVALID_STATE){
+    if (err == ESP_ERR_INVALID_STATE)
+    {
```

Comment:
Please change back the styling. You may consider running [format.sh](https://github.com/espressif/esp-idf/blob/master/tools/format.sh) on new files.